### DISCUSSION TITLE
Underestimated uncertainties and sigma_lnf

### DISCUSSION THREAD

**Discussion by dmrowan** [phoebe-project/phoebe2#895] [Questions]
Hello, 

I'm using PHOEBE to fit a light curve and RV dataset of a detached EB. My RV uncertainties are likely underestimated, so I'd like to sample over sigmas_lnf when I run MCMC.

I am confused about the implementation of sigmas_lnf in PHOEBE. The [documentation](http://phoebe-project.org/docs/2.4/api/phoebe.parameters.ParameterSet.calculate_lnlikelihood) suggests that the sigmas are increased by 
`interpolated_model**2 * np.exp(2 * sigmas_lnf)`

Why does this term include the interpolated model? For example, if I have two identical systems and one has vgamma=10km/s and the other has vgamma=100 km/s, why would the RV uncertainties be rescaled to larger values in the later case? I would also think that the RV observations should know nothing about the phase of the binary at the observation time. 

Reading through the [emcee line fitting tutorial](https://emcee.readthedocs.io/en/stable/tutorials/line/) that is linked in the [emcee sampler example](http://phoebe-project.org/docs/2.4/tutorials/emcee), it seems like their likelihood function adds `model**2 * np.exp(2 * sigmas_lnf)` to the "measured" uncertainties because they've intentionally set up an example where the uncertainties are underestimated more at larger y values. 

How would you recommend using the sigmas_lnf parameter for an RV dataset?

--Dom

**Comment by kecnry** [comment]
We've actually been looking into this already this week and @kmhambleton opened #901 to change the behavior for RVs.  Are the changes there in agreement with what you'd expect?

**Reply by dmrowan** [reply]
Yea, I think that makes sense. This seems to match up with how TheJoker adds a stellar jitter term ([eqn 10 here](https://arxiv.org/abs/1610.07602)).
